main/dnsmasq/APKBUILD

```diff
@@ -1,50 +1,58 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=dnsmasq
 pkgver=2.76
-pkgrel=0
+pkgrel=1
 pkgdesc="Small forwarding DNS server"
 url="http://www.thekelleys.org.uk/dnsmasq/"
 arch="all"
 license="GPL2"
-depends=
+depends=""
 makedepends="linux-headers"
 install="$pkgname.pre-install $pkgname.pre-upgrade"
 subpackages="$pkgname-doc"
 source="http://www.thekelleys.org.uk/dnsmasq/$pkgname-$pkgver.tar.gz
 	$pkgname.initd
 	$pkgname.confd
 	"
+builddir="$srcdir/$pkgname-$pkgver"
 
 build() {
-	cd "$srcdir/$pkgname-$pkgver"
+	cd "$builddir"
+
 	sed -i -e 's/^AWK.*/AWK = awk/' Makefile
 	make PREFIX=/usr \
 		CFLAGS="$CFLAGS" \
 		DBUS_LIBS= \
 		DBUS_CFLAGS= \
 		DBUS_MINOR= \
-		all || return 1
```

Comment:
We already discussed this in past. Having explicit return is better than implicit one.